lection6/task2.js

```diff
@@ -0,0 +1,71 @@
+// 1. У вас есть массив названий пицц вашего конкурента. Создайте функцию, которая будет принимать ваш набор названий пицц (массив) 
+//   и возвращать только те, которых нет у конкурента (тоже массив). Если все ваши пиццы есть у конкурента - вернуть null
+//   Пиццы конкурента:
+//   const competitorPizzas = ['Peperoni', 'Caprichosa', 'Diablo', '4 cheeses', 'hawai'];
+
+function isMyPizzaUnique(myPizzas) {
```

Comment:
Add validation for passed argument
Good practice is to convert pizza names to lower case and compare them